### PR TITLE
Implement escape feature

### PR DESCRIPTION
**Pull request by DaisukeDaisuke**
> I use machine translation

Implemented the ability to add a space after a letter
The current default is to insert a space after `@` even when embedding is enabled

### CODE REVIEW

File: resources/config.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ messages:
         fields: []
     to_discord_channels:
       - "123456789"
+    escapes:
+      - "@"
```
Comment (DaisukeDaisuke) [inline resources/config.yml:35]: If escaping is to be disabled, the value must be changed to`: []`, so confusion can be avoided by using`: ["@"]` by default.
```suggestion
    escapes: ["@"] 
```

File: resources/help/english.txt [hunk 2]
```diff
@@ -72,6 +72,8 @@ messages:                                               <- Messages section allo
       - "12345678912345678"                             <- Discord channel IDs
       - "42356325325312412"
 
+    escapes:                                             <- List of characters to escape "{MESSAGE}", to disable, delete the "- @" line and replace it with "escapes: []".
+      - "@"                                              <- A space is added after every '@'
```
Comment (DaisukeDaisuke) [inline resources/help/english.txt]:
```suggestion
    escapes: ["@"]                                      <- List of characters to escape {MESSAGE}, with a space inserted after the "@" by default.
```

File: resources/help/english.txt [hunk 3]
```diff
@@ -72,6 +72,8 @@ messages:                                               <- Messages section allo
       - "12345678912345678"                             <- Discord channel IDs
       - "42356325325312412"
 
+    escapes:                                            <- List of characters to escape "{MESSAGE}", to disable, delete the "- @" line and replace it with "escapes: []".
+      - "@"                                             <- A space is added after every '@'
```
Comment (DaisukeDaisuke) [inline resources/help/english.txt:76]:
```suggestion
    escapes: ["@"]                                      <- List of characters to escape {MESSAGE}, with a space inserted after the "@" by default.
```

**Comment by JaxkDev** [comment]
Sorry I've never seen this PR appear before, Must not have notifications at all over here.

This seems very very specific to just mentions I can't think of another usecase...
If that's the case I'd rather go down the route of allowing/disallowing mentions via discord api

**Comment by JaxkDev** [comment]
Thanks for the PR but this over engineers a solution, since the only use case I can still think of is mentions we'll use the mention parameter when sending a message in next version (once DiscordBot 3.x is out)

**Pull request closed without merging**